_posts/0000-01-05-BinduTuli0609.md

```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+Is the training going too slow or are we on track?
```

Comment:
Hmm, that's a question for the training guides.

Reply:
It is a little slower 🐢 but we aren't in danger 💀 of missing any content 😺